src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2468,12 +2399,11 @@ void GetGroundHeatExchangerInput(EnergyPlusData &state)
         ShowContinueError(state, "Simulation indicated these objects were found, but input processor doesn't find any");
         ShowContinueError(state, "Check inputs for GroundHeatExchanger:System and GroundHeatExchanger:Slinky");
         ShowContinueError(state, "Also check plant/branch inputs for references to invalid/deleted objects");
-        errorsFound = true;
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2471]:(style),[unreadVariable],Variable 'errorsFound' is assigned a value that is never used.